elastic/kibana
# [Cloud Security] Bug fix - show origin event's with primary color instead of danger

## Summary

Bug description:

**Actual:** The node's color is red when exploring events through Explore or Timeline. 
**The expected** color of events is blue.

Before:

![385007418-f0a6bd7e-dbc9-43ad-99b8-a07bcad85075](https://github.com/user-attachments/assets/7bf198f3-9a32-4d27-84db-3e97b5bf312b)

After:


https://github.com/user-attachments/assets/f1a10deb-65f5-43be-a351-6fca34f855cb

https://github.com/user-attachments/assets/223534f4-09a2-4b41-85bc-c2195dd153ba

**How to test this PR:**

- Enable the feature flag 

`kibana.dev.yml`:

```yaml
uiSettings.overrides.securitySolution:enableVisualizationsInFlyout: true
xpack.securitySolution.enableExperimental: ['graphVisualizationInFlyoutEnabled']
```

- Load mocked data:

```bash
node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/logs_gcp_audit \ 
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601

node scripts/es_archiver load x-pack/test/cloud_security_posture_functional/es_archives/security_alerts \
  --es-url http://elastic:changeme@localhost:9200 \
  --kibana-url http://elastic:changeme@localhost:5601
```

- Make sure you include data from Oct 13 2024. (in the video I use Last year)


### Checklist
